sdk/tests/js/webgl-test-utils.js

```diff
@@ -39,24 +39,30 @@ var loggingOff = function() {
  * @param {number} value The enum value.
  * @return {string} The enum as a string.
  */
-var glEnumToString = function(gl, value) {
-  if (gl.NO_ERROR === undefined)
-    throw new Error('glEnumToString: `gl` must be a WebGL2?RenderingContext');
+var glEnumToString = function(glOrExt, value) {
   if (value === undefined)
     throw new Error('glEnumToString: `value` must not be undefined');
 
-  // Optimization for the most common enum:
-  if (value === gl.NO_ERROR) {
-    return "NO_ERROR";
-  }
-  for (var p in gl) {
-    if (gl[p] == value) {
-      if (p == 'drawingBufferWidth' || p == 'drawingBufferHeight') {
+  if (glOrExt.keyByValue === undefined) {
+    const keyByValue = glOrExt.keyByValue = {};
```

Comment:
keyByValue could be associated with the prototype instead of the object, I think